Makefile

```diff
@@ -1,8 +1,15 @@
 CFLAGS += -Wall -Wextra
+VERSION = r$(shell git rev-list --count HEAD)-$(shell git rev-parse --short=8 HEAD)
 
 all: uradvd
 uradvd: uradvd.o
 
+uradvd.o:
```

Comment:
Dependency `uradvd.c` is missing from the rule

Reply:
resolved in 84caf39f28b3a3589605f852f1b9772888c5de53